src/sql/workbench/contrib/notebook/browser/notebook.contribution.ts

```diff
@@ -173,6 +173,15 @@ configurationRegistry.registerConfiguration({
 			'type': 'boolean',
 			'default': true,
 			'description': localize('notebook.sqlStopOnError', "SQL kernel: stop Notebook execution when error occurs in a cell.")
+		},
+		'notebook.trustedNotebooks': {
```

Comment:
This name doesn't really match what's actually stored here, suggest `notebook.trustedNotebookDirectories` instead